fdbclient/BackupAgentBase.actor.cpp

```diff
@@ -549,8 +549,12 @@ ACTOR Future<Void> readCommitted(Database cx,
 				tr.setOption(FDBTransactionOptions::ACCESS_SYSTEM_KEYS);
 			if (lockAware)
 				tr.setOption(FDBTransactionOptions::LOCK_AWARE);
-			if (CLIENT_KNOBS->ENABLE_REPLICA_CONSISTENCY_CHECK_ON_BACKUP_READS)
+			if (CLIENT_KNOBS->ENABLE_REPLICA_CONSISTENCY_CHECK_ON_BACKUP_READS) {
 				tr.setOption(FDBTransactionOptions::ENABLE_REPLICA_CONSISTENCY_CHECK);
+				int64_t requiredReplicas = CLIENT_KNOBS->CONSISTENCY_CHECK_REQUIRED_REPLICAS;
+				tr.setOption(FDBTransactionOptions::CONSISTENCY_CHECK_REQUIRED_REPLICAS,
```

Comment:
fdbclient/vexillographer/fdb.options specify this option of type `Int`, but here int64_t is used, which doesn't seem to be consistent.

Reply:
Good question! Using "Int-type" was causing  "Transaction::setOption()" (https://github.com/apple/foundationdb/blob/bf5321855657e485b04274dfea303667643351eb/fdbclient/NativeAPI.actor.cpp#L7016) to throw here https://github.com/apple/foundationdb/blob/bf5321855657e485b04274dfea303667643351eb/fdbclient/NativeAPI.actor.cpp#L2095. I am not sure how the other "Int-type" option value calls are working in that method.

Reply:
Looks like `Int` means int64_t, same applies to other options.